### PR TITLE
Fix code coverage

### PR DESCRIPTION
**Pull request by jillrdoty**
Getting code coverage back up so checks pass.

### CODE REVIEW

**Review by shelleyMurphy — APPROVED** [review]

File: cohort-util/src/test/java/com/ibm/cohort/valueset/ValueSetUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	@Test
+	public void testUnsuppliedUrl(){
+		thrown.expect(IllegalArgumentException.class);
+		thrown.expectMessage("URL must be supplied");
+		ValueSetArtifact artifact = new ValueSetArtifact();
+		ValueSetUtil.validateArtifact(artifact);
+	}
+
+	@Test
+	public void testUnsuppiedIFhirResource(){
```
Comment (msargentibm) [inline cohort-util/src/test/java/com/ibm/cohort/valueset/ValueSetUtilTest.java:39]: Typo  in  several test names: `Unsuppied` -> `Unsupplied`

File: cohort-util/src/main/java/com/ibm/cohort/valueset/CodeSystemLookup.java [hunk 2]
```diff
@@ -9,8 +9,14 @@
 package com.ibm.cohort.valueset;
 
 
+import com.ibm.cohort.annotations.Generated;
+
 //taken from cqf-tooling's CodeSystemLookupDictionary.java
+//tagged to avoid coverage checks
+@Generated
```
Comment (msargentibm) [inline cohort-util/src/main/java/com/ibm/cohort/valueset/CodeSystemLookup.java:16]: If the class is tagged with `@Generated`, then do we need `cohort-util/src/test/java/com/ibm/cohort/valueset/CodeSystemLookupTest.java`?

**Review by csandersdev — COMMENTED** [review]
LGTM

**Review by msargentibm — COMMENTED** [review]
Saw some typos  and had a question about a use of the `@Generated` annotation.

File: cohort-util/src/main/java/com/ibm/cohort/valueset/CodeSystemLookup.java (hunk 2 above)
Comment (jillrdoty) [reply cohort-util/src/main/java/com/ibm/cohort/valueset/CodeSystemLookup.java:16]: The test was added before the generated tag was added... it feels silly to test all the cases of the switch statement, but if you don't it significantly drags down the coverage.  I'm ok deleting the test, but it also doesn't feel bad to have around.

Comment (jillrdoty) [reply cohort-util/src/main/java/com/ibm/cohort/valueset/CodeSystemLookup.java:16]: The code isn't technically generated, but we don't have another way of ignoring classes for test coverage purposes.

Comment (msargentibm) [reply cohort-util/src/main/java/com/ibm/cohort/valueset/CodeSystemLookup.java:16]: > The code isn't technically generated, but we don't have another way of ignoring classes for test coverage purposes.

Understood. As long as including both the `@Generated` annotation and the test class was done on purpose then that works for me. I just wanted to make sure there wasn't a cleanup step missed after the annotation was added.

**Review by msargentibm — APPROVED** [review]

**Pull request merged**